### PR TITLE
pin tox dependency virtualenv to 16.7.9 to fix travis build failures

### PR DESCRIPTION
**Pull request by lyw07**
### Summary

This is to fix our travis build failures that started today due to the `tox`'s dependency `virtualenv` new release. 
This should be a temporary workaround before virtualenv fixes the issue on their side and has a new release with the fix.
(will backport this to 0.13.x release branch)

### Reviewer guidance

does the travis build pass?

### References

https://travis-ci.org/learningequality/kolibri/jobs/648487308
https://github.com/tox-dev/tox/issues/1516

----

### Contributor Checklist

PR process:

- [X] PR has the correct target branch and milestone
- [X] PR has 'needs review' or 'work-in-progress' label
- [X] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Review by rtibbles — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ before_install:
   - git describe --tags --always
   # Required until this is fixed: https://github.com/travis-ci/travis-ci/issues/9112
   - sudo apt-get update --option Acquire::Retries=100 --option Acquire::http::Timeout="60"
-  - pip install -U pip codecov tox
+  - pip install -U pip codecov virtualenv==16.7.9 tox
```
Comment (gaborbernat) [inline .travis.yml:98]: what was the error message you were getting?

Comment (lyw07) [reply .travis.yml:98]: Hi! We got the error in our Travis CI builds, so we don't get to see the full traceback of the error. But here is the error message we got:
```
ERROR:root:ImportError: cannot import name 'ensure_text'
ERROR: InvocationError for command /home/travis/virtualenv/python3.6.3/bin/python -m virtualenv --no-download --python /home/travis/virtualenv/python3.6.3/bin/python pythonlint3 (exited with code 1)
```

We reached out to Travis CI support team. They helped us by running a diff check of builds and found that the only difference was the version of virtualenv. 
Please feel free to let us know if you need any additional information. Thank you!

Comment (gaborbernat) [reply .travis.yml:98]: I think installing six>=1.12 would have also solved your issue, even with the latest virtualenv. Though you might want to wait for https://github.com/pypa/virtualenv/pull/1578 to land before you switch.

Comment (lyw07) [reply .travis.yml:98]: thank you for your suggestion! we used this PR as a temporary workaround just to get our builds pass and will wait for the virtualenv change as you suggested :)

**Comment by DXCanas** [comment]
Might we have to backport this to previous kolibri versions? like 0.10 - 0.12?

**Comment by lyw07** [comment]
I don't think so since we don't currently have builds running there, and the error will be resolved once the dependency gets fixed.

**Pull request merged**